st2auth_ldap_backend/ldap_backend.py

```diff
@@ -35,7 +35,9 @@ class LDAPAuthenticationBackend(object):
         * Bind Distinguish Name with user lookup.
         * Bind Distinguish Name with user and group lookup.
     """
-    def __init__(self, ldap_uri, use_tls=False, bind_dn='', bind_pw='', user=None, group=None):
+    def __init__(self, ldap_uri, use_tls=False, bind_dn='',
+                 bind_pw='', user=None, group=None,
+                 chase_referrals=True):
```

Comment:
Please update the configuration section in README to include the chase_referrals setting.